packages/shared/src/features/onboarding/hooks/useFunnelNavigation.ts

```diff
@@ -67,19 +69,21 @@ function updateURLWithStepId({
 }) {
   const params = new URLSearchParams(searchParams.toString());
   params.set('stepId', stepId);
-  router.replace(`/${pathname}?${params.toString()}`, { scroll: false });
+  router.push(`${pathname}?${params.toString()}`, { scroll: true });
```

Comment:
Why are we doing this change?

Reply:
I was moving a step into the routing with shallow active, but for now I don't think we are going forward with this so I can revert it.

Reply:
let's keep it. Nimrod told me he wants back button support